components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/IdPGroup.java

```diff
@@ -48,6 +48,10 @@ public class IdPGroup implements Serializable {
     @XmlTransient
     @JsonIgnore
     private String idpGroupId = null;
+    @IgnoreNullElement
+    @XmlTransient
+    @JsonIgnore
+    private String idpId = null;
```

Comment:
need to update wsdl as well

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5143/commits/59d76c136ee1265884f58566d006288cb4d3dbde